robocop/checkers/duplications.py

```diff
@@ -88,10 +98,24 @@ def check_duplicates(self, container, rule):
     def visit_TestCase(self, node):  # noqa
         testcase_name = normalize_robot_name(node.name)
         self.test_cases[testcase_name].append(node)
+        self.generic_visit(node)
 
     def visit_Keyword(self, node):  # noqa
         keyword_name = normalize_robot_name(node.name)
         self.keywords[keyword_name].append(node)
+        self.generic_visit(node)
+
+    def visit_KeywordCall(self, node):  # noqa
+        assign = node.get_tokens(Token.ASSIGN)
+        if not assign:
+            return
```

Comment:
I think this is not needed because if it is empty, then the for loop below will not be executed